test/integration/init_int_test.go

```diff
@@ -60,11 +60,20 @@ func (suite *InitIntegrationTestSuite) runInitTest(addPath bool, lang string, ex
 		computedArgs = append(computedArgs, ts.Dirs.Work)
 	}
 
+	env := []string{}
+	if sourceRuntime {
+		env = append(env, constants.DisableRuntime+"=false")
```

Comment:
I see this is targeted at v45, so when you merge this into v46, I think we'll want to flip the conditional (`if !sourceRuntime { env = append(env, constants.DisableRuntime+"=true") }`)

Reply:
Will update it in the runtime refactor PR.